ur_robot_driver/src/ur/ur_driver.cpp

```diff
@@ -141,8 +144,11 @@ ur_driver::UrDriver::UrDriver(const std::string& robot_ip, const std::string& sc
 
 std::unique_ptr<rtde_interface::DataPackage> ur_driver::UrDriver::getDataPackage()
 {
-  std::chrono::milliseconds timeout(100);  // We deliberately have a quite large timeout here, as the robot itself
-                                           // should command the control loop's timing.
+  std::chrono::milliseconds timeout(get_packet_timeout_);  // This can take one of two values, 0ms or 100ms. The large
+                                                           // timeout is for when the robot is commanding the control
+                                                           // loop's timing (read is blocking). The zero timeout is for
+                                                           // when the robot is sharing a control loop with
+                                                           // something else (combined_robot_hw)
```

Comment:
We could move the comment above the line to improve readability.

Reply:
How's this? https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/86/commits/529c711c22bb58167a7c5b195dbffc3dd038b2c9

Reply:
Like that? https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/86/commits/4cf8d6fc8d8c8240008c319659777e5f55d546b6

Reply:
Like that? https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/pull/86/commits/4cf8d6fc8d8c8240008c319659777e5f55d546b6